src/render/2d/kekule.render.canvasRenderer.js

```diff
@@ -491,6 +491,8 @@ Kekule.Render.CanvasRendererBridge = Class.create(
 			//if (context.lineWidth !== 1)
 			context.lineWidth = 1;
 		}
+		if (options.strokeColor && options.strokeColor == "#999999")
```

Comment:
Why are we setting the strokeColor as a check?

Reply:
Instead of just calling it out? Why wrap it in an `if` statement?
```suggestion
		options.strokeColor = "#686868";
```

Reply:
@AaronLlanos The reason why I am using `strokeColor` as a check is because color of reaction arrows is set using `strokeColor`. 
Also, the reason I am wrapping it inside the `if` statement is because the color of all structures other than Reaction Arrows is `#000000` rather than `#999999`. We only need to change the color of Reaction Arrows as per Phil's comment.